File: S09-google-admob/CL06-GoogleAdmob/app/src/main/java/com/androidavanzado/googleadmob/MainActivity.java
package com.androidavanzado.googleadmob;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.google.android.gms.ads.MobileAds;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        MobileAds.initialize(this, "ca-app-pub-3850129240631389~9209579640");
    }
}

File: S03-conceptos-basicos/CL02-CicloVidaActivity/app/src/main/java/com/androidavanzado/cl02_ciclovidaactivity/MainActivity.kt
package com.androidavanzado.cl02_ciclovidaactivity

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Log.i("LOGCICLO", "Entramos en método onCreate")
    }

    override fun onStart() {
        super.onStart()
        Log.i("LOGCICLO", "Entramos en método onStart")

    }

    override fun onResume() {
        super.onResume()
        Log.i("LOGCICLO", "Entramos en método onResume")
    }

    override fun onPause() {
        super.onPause()
        Log.i("LOGCICLO", "Entramos en método onPause")

    }

    override fun onStop() {
        super.onStop()
        Log.i("LOGCICLO", "Entramos en método onStop")

    }

    override fun onDestroy() {
        super.onDestroy()
        Log.i("LOGCICLO", "Entramos en método onDestroy")

    }
}
